### PR TITLE
Implemented the LoadBanList method. Closed #48

### LINKED ISSUE #48: Implement the LoadWordBanList method
Implement the LoadWordBanList method in the WordBanListService class. Add different loading options: from file and from server.

### PR DESCRIPTION
**Pull request by HecuseS**

### CODE REVIEW

File: Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/WordBanListServiceTest.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.VisualStudio.TestPlatform.Utilities;
+using user_monitoring_gui.Models;
+using user_monitoring_gui.Services;
+using user_monitoring_gui.Services.Interfaces;
+using Xunit.Abstractions;
```
Comment (TopAcademy2022) [inline Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/WordBanListServiceTest.cs]: Remove this line.

File: Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/WordBanListServiceTest.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.VisualStudio.TestPlatform.Utilities;
```
Comment (TopAcademy2022) [inline Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/WordBanListServiceTest.cs]: Remove this line.

File: Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/WordBanListServiceTest.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+using Xunit.Abstractions;
+
+namespace user_monitoring_gui_tests
+{
+    /*!
+    * \ brief Class  WordBanListServiceTest for testing methods 
+    */
+    public class WordBanListServiceTest
+    {
+        private static readonly IServerRequest request; // Itialization of the required parameter
```
Comment (TopAcademy2022) [inline Tests/user-monitoring-gui-tests/user-monitoring-gui-tests/WordBanListServiceTest.cs]: Switch comment style to Doxygen style.

File: user-monitoring-gui/Services/ConsoleUserInterface.cs
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/ConsoleUserInterface.cs:1]: Revert this file to main (HEAD).

File: user-monitoring-gui/Services/Interfaces/IServerRequest.cs
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/Interfaces/IServerRequest.cs:1]: Revert this file to main (HEAD).

File: user-monitoring-gui/Services/RunProgramReader.cs
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/RunProgramReader.cs:1]: Revert this file to main (HEAD).

File: user-monitoring-gui/Services/WordBanListService.cs [hunk 4]
```diff
@@ -1,4 +1,5 @@
 ﻿using System.IO;
+using System.Security.Cryptography.X509Certificates;
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/WordBanListService.cs]: Remove this line.

File: user-monitoring-gui/Services/WordBanListService.cs [hunk 5]
```diff
@@ -14,7 +15,9 @@ public class WordBanListService : IWordBanListService
     {
         /* ! interface IServerRequest variable initialization  */
 
-        private IServerRequest _serverReqwest; 
+        private IServerRequest _serverReqwest;
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/WordBanListService.cs]: Rename "_serverReqwest" to correct name "_serverRequest".

File: user-monitoring-gui/Services/WordBanListService.cs [hunk 6]
```diff
@@ -14,7 +15,9 @@ public class WordBanListService : IWordBanListService
...
+
+
```
Comment (TopAcademy2022) [inline user-monitoring-gui/Services/WordBanListService.cs]: Why space line?

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]
Refactoring...

**Review by cany8042 — DISMISSED** [review]

**Pull request closed without merging**